<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>8</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Каталоги</string>
    </attribute>
    <widget class="QPushButton" name="pbDirPrograms">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>11</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Программы</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leDirPrograms">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>14</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbDirData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Данные</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leDirData">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="chbSaveData">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>50</y>
       <width>81</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранять</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Порт</string>
    </attribute>
    <widget class="QPushButton" name="pbUpdate">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>13</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbPort">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>14</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Разные</string>
    </attribute>
    <widget class="QPushButton" name="pbColorDisplay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Цвет дисплея</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbColorSimbols">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>47</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Цвет символов</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
